osu.Desktop.VisualTests/Tests/TestCasePlayer.cs

```diff
@@ -18,41 +21,54 @@ namespace osu.Desktop.VisualTests.Tests
 {
     class TestCasePlayer : TestCase
     {
+        private WorkingBeatmap beatmap;
         public override string Name => @"Player";
 
         public override string Description => @"Showing everything to play the game.";
 
+        [BackgroundDependencyLoader]
+        private void load(BeatmapDatabase db)
+        {
+            beatmap = db?.GetWorkingBeatmap(db.Query<BeatmapInfo>().Where(b => b.Mode == PlayMode.Osu).FirstOrDefault());

```

Comment:
db will never be null unless some weird shit is going on. BackgroundDependencyLoader accepts permitNulls as a parameter, which is false by default, and will throw rather than try to initialize something it can't fulfill a dependency of.